layouts/partials/search.html

```diff
@@ -5,12 +5,5 @@
     <script type="text/javascript" src="{{"js/lunr.min.js" | relURL}}"></script>
     <script type="text/javascript" src="{{"js/auto-complete.js" | relURL}}"></script>
     <link href="{{"css/auto-complete.css" | relURL}}" rel="stylesheet">
-    <script type="text/javascript">
```

Comment:
This was breaking, since it wasn't actually getting Hugo-rendered properly. Assumed that removing a broken thing won't break anything that was working.